intellij/src/org/scalafmt/intellij/ScalafmtAction.scala

```diff
@@ -65,19 +65,29 @@ class ScalafmtAction extends AnAction {
     }
   }
 
+  private def getConfigFileInPath(path: String) = {
+    val file = FileOps.getFile(path, ".scalafmt")
+    if (file.isFile)
+      Some(file.getAbsolutePath)
+    else
+      None
+  }
+
+  private val homeDir = System.getProperty("user.home")
+
   private def getStyle(event: AnActionEvent): ScalafmtStyle = {
     val customStyle = for {
       project <- Option(event.getData(CommonDataKeys.PROJECT))
-      configFile = FileOps.getFile(project.getBasePath, ".scalafmt")
-          if configFile.isFile
-      style <- StyleCache.getStyleForFile(configFile.getAbsolutePath)
+      localConfig = getConfigFileInPath(project.getBasePath)
+      globalConfig = getConfigFileInPath(homeDir)
+      configFile <- localConfig orElse globalConfig
```

Comment:
I try to avoid infix application for non-symbolic operators: `localConfig.orElse(globalConfig)`